src/pfe/Dockerfile_ppc64le

```diff
@@ -0,0 +1,219 @@
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v20.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+#FROM centos as codewind-pfe-base
+
+#FROM centos@sha256:f94c1d992c193b3dc09e297ffd54d8a4f1dc946c37cbeceb26d35ce1647f88d9 as codewind-pfe-base
+FROM centos@sha256:b5e66c4651870a1ad435cd75922fe2cb943c9e973a9673822d1414824a1d0475 as codewind-pfe-base
+LABEL org.opencontainers.image.title="Codewind-PFE" org.opencontainers.image.description="Codewind PFE" \
+      org.opencontainers.image.url="https://codewind.dev/" \
+      org.opencontainers.image.source="https://github.com/eclipse/codewind"
+
+#ARG BUILDAH_RPM=https://cbs.centos.org/kojifiles/packages/buildah/1.11.4/4.el7/x86_64/buildah-1.11.4-4.el7.x86_64.rpm
+#ARG SLIRP=https://cbs.centos.org/kojifiles/packages/slirp4netns/0.3.0/2.git4992082.el7/x86_64/slirp4netns-0.3.0-2.git4992082.el7.x86_64.rpm
+ARG SLIRP=https://rpmfind.net/linux/opensuse/ports/ppc/distribution/leap/15.2/repo/oss/ppc64le/slirp4netns-0.3.0-lp152.1.3.ppc64le.rpm
+#ARG LIBSEC=https://cbs.centos.org/kojifiles/packages/libseccomp/2.4.1/0.el7/x86_64/libseccomp-2.4.1-0.el7.x86_64.rpm
+
+# Download the buildah RPM
+#RUN curl -f -o buildah.rpm $BUILDAH_RPM
+RUN curl -f -o slirp4netns.rpm $SLIRP
+#RUN curl -f -o libseccomp.rpm $LIBSEC
+
+# Download and set up Node 10.x RPM
+#RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash -
+
+RUN yum install -y epel-release \
+    && yum -y --enablerepo=epel install zip unzip sudo ca-certificates openssl  wget gcc-c++ make slirp4netns.rpm 
+
+
+RUN wget https://nodejs.org/download/release/latest-v13.x/node-v13.5.0-linux-ppc64le.tar.gz \
+    && tar xvzf  node-v13.5.0-linux-ppc64le.tar.gz
+ENV PATH=/node-v13.5.0-linux-ppc64le/bin:$PATH
+
+RUN yum install -y  --enablerepo=epel libseccomp buildah \
```

Comment:
The version of buildah bundled by default with CentOS is too backlevel for us to use, which is why we need to use ` ARG BUILDAH_RPM=https://cbs.centos.org/kojifiles/packages/buildah/1.11.4/4.el7/x86_64/buildah-1.11.4-4.el7.x86_64.rpm` on x86. 

We might need to work with the buildah team to an get up-to-date Buildah RPM for PPC.

Reply:
Yes, I tried using buildah 1.11.2 rpm for ppc64le, but does not work. It keeps on throwing missing dependency error for one or the other.